cupyx/fallback_mode/utils.py

```diff
@@ -41,3 +44,78 @@ def _call_numpy(func, args, kwargs):
     cupy_res = data_transfer._get_cupy_result(numpy_res)
 
     return cupy_res
+
+
+_thread_locals = threading.local()
+_thread_locals.dispatch_type = 'warn'
+
+FallbackWarning = type('FallbackWarning', (Warning,), {})
+warnings.simplefilter(action='always', category=FallbackWarning)
+
+
+def geterr():
+    return _thread_locals.dispatch_type
+
+
+def seterr(new_dispatch):
+
+    old = _thread_locals.dispatch_type
+
+    if new_dispatch in ['print', 'warn', 'log', 'ignore', 'raise']:
+        _thread_locals.dispatch_type = new_dispatch
+        return old
+
+    raise ValueError('{} is not valid dispatch type'.format(new_dispatch))
+
+
+class FallbackLogger:
+
+    logger = None
+
+    @classmethod
+    def setlogger(cls, logger):
+        cls.logger = logger
+
+    @classmethod
+    def getlogger(cls):
+
+        if cls.logger is None:
+            raise AttributeError('Logger not initiated')
+
+        return cls.logger
+
+
+def dispatch_notification(func):
+
+    msg = "'{}' method not in cupy, falling back to '{}.{}'".format(
+        func.__name__, func.__module__, func.__name__)
+
+    raise_msg = "'{}' method not found in cupy".format(
+        func.__name__)
+
+    if _thread_locals.dispatch_type == 'print':
+        print("Warning: {}".format(msg))
+
+    elif _thread_locals.dispatch_type == 'warn':
+        warnings.warn(msg, FallbackWarning, stacklevel=3)
+
+    elif _thread_locals.dispatch_type == 'log':
+        logger = FallbackLogger.getlogger()
+        logger.warning(msg)
+
+    elif _thread_locals.dispatch_type == 'raise':
+        raise AttributeError(raise_msg)
+
```

Comment:
How about adding the following line for an error handling?
```
    else:
        raise RuntimeError('This line is never reached.')
```

Reply:
Okay, will need to add `pass` for `ignore` then.

Reply:
On second thought, I think this is not needed as L68 will make it compulsory to satisfy at least one if statement.

Reply:
Yes, this line is never reached no matter how the users use this feature. This line is for us being able to debug easily.

Reply:
Sorry, `assert False` is appropriate.

Reply:
So, 
```python
else:
    assert False
```
Right?

Reply:
Yes!

Reply:
Done!